examples/lb3-application/README.md

```diff
@@ -103,6 +103,242 @@ Start the new LB4 application
 Open the URL printed to console to view your front-end served from `public`
 directory or go to `http://127.0.0.1:3000/explorer` to explore the REST API.
 
+The LB3 application is now successfully mounted on a LB4 application, but we
+can further optimize the setup so that only the bare neccessary artifacts
+from the LoopBack 3 application remain. This includes moving almost all of the
+middleware to a common location so that they are shared by both the LoopBack 3
+and the LoopBack 4 apps.
+
+## Migrating Express middleware from LB3 app
+
+First off, edit the LB3 app's `config.json` file.
```

Comment:
I'd prefer to use point form on the steps, similar to what we have in the previous section.  The goal is to show clearly what are the steps involved, so I find having numbers on the steps would help a lot.

Reply:
Done.